src/legacy/ui/public/index_patterns/_index_pattern.js

```diff
@@ -119,24 +117,35 @@ export function IndexPatternProvider(Private, config, Promise, confirmModalPromi
     }
 
     if (indexPattern.isUnsupportedTimePattern()) {
-      if (!isUserAwareOfUnsupportedTimePattern(indexPattern)) {
-        const warningTitle = i18n.translate('common.ui.indexPattern.warningTitle', {
-          defaultMessage: 'Support for time intervals was removed',
-        });
+      const warningTitle = i18n.translate('common.ui.indexPattern.warningTitle', {
+        defaultMessage: 'Support for time interval index patterns removed',
+      });
 
-        const warningText = i18n.translate('common.ui.indexPattern.warningText', {
-          defaultMessage: 'For more information, view the ["{title}" index pattern in management]({link})',
-          values: {
-            title: indexPattern.title,
-            link: kbnUrl.getRouteHref(indexPattern, 'edit'),
-          },
-        });
+      const warningText = i18n.translate('common.ui.indexPattern.warningText', {
+        defaultMessage: 'Instead of querying indices only matching the pattern {title}, you are now querying all indices ' +
+          'matching {index}. This index pattern should be migrated to a wildcard-based index pattern. To do so, edit the ' +
+          'index pattern in Management.',
```

Comment:
UX nit: This message is hard to read before the notification disappears. I wonder if we could wordsmith it to make things shorter, e.g.

```
Support for time interval index patterns removed

Currently querying all indices matching {index}.
{title} should be migrated to a wildcard-based index pattern.

Edit >
```